partiql-planner/src/main/kotlin/org/partiql/planner/transforms/RelConverter.kt

```diff
@@ -96,7 +90,11 @@ internal object RelConverter {
             }
             // SELECT VALUE ... FROM
             is Select.Value -> {
-                val constructor = projection.constructor.toRex(env)
+                val projectionOp = rel.op as? Rel.Op.Project ?: error("SELECT VALUE should have a PROJECT underneath")
+                assert(projectionOp.projections.size == 1) {
+                    "Expected SELECT VALUE projection to have a single binding. However, it looked like ${projectionOp.projections}"
+                }
+                val constructor = rex(StaticType.ANY, rexOpVarResolved(0))
```

Comment:
I don't think it necessarily needs a PROJECT rel.op, but the input rel.op (regardless of what it is) must output exactly one binding.